docs/input.md

```diff
@@ -166,22 +166,21 @@ How does Skywalker know this isn't a list containing the temperatures 230.15 K,
 300.15 K, and 1 K? It uses a set of simple rules to determine how to interpret
 these values.
 
-A list with values `[v1, v2, v3]` that satisfies the following properties is
-expanded into a uniformly spaced set of values between `v1` and `v2` with
-spacing `v3`:
+A list with values $\[v_1, v_2, v_3\]$ that satisfies the following properties is
+expanded into a uniformly spaced set of values between $v_1$ and $v_2$ with
+spacing $v_3$:
 
-* `v1 < v2` and `v3 < v2`, **OR**
-* `v1 < v2 < 0` and `|v3| < |v2 - v1|`.
+* $v_1 < v_2$
+* $0 < v_3 < v_2$ **or** $v_2 < 0$ and $0 < v_3 < (v_2 - v_1)/2$
 
 All other lists are interpreted as lists containing 3 values.
 
```

Comment:
A few examples here would be helpful:

```
list [0, 1, 0.1] expands to [0.  0.1 0.2 0.3 0.4 0.5 0.6 0.7 0.8 0.9 1. ]
list [-2, 0, 0.2] does not expand
list [-2, 1, 0.25] expands to [-2.   -1.75 -1.5  -1.25 -1.   -0.75 -0.5  -0.25  0.    0.25  0.5   0.75
  1.  ]
list [-10, -1, 1] expands to [-10.  -9.  -8.  -7.  -6.  -5.  -4.  -3.  -2.  -1.]
list [-11, 1, 1] does not expand
list [-11, 1, 2] does not expand
list [-7, 2, 1] expands to [-7. -6. -5. -4. -3. -2. -1.  0.  1.  2.]
```

btw: the above is the output from this python script:

```
import numpy as np

def cond1(l):
  return (0 < l[2] and l[2] < l[1])

def cond2(l):
  return (l[1]<0 and 0 < l[2] and l[2] < (l[1] - l[0]))

def list_expand(l):
  result = np.array(l)
  if len(l) == 3:
    if l[0] < l[1]:
      if cond1(l) or cond2(l):
        result = np.linspace(l[0], l[1], int(np.ceil((l[1]-l[0])/l[2]))+1)
  return result

if __name__ == '__main__':
  lists = [ [0, 1, 0.1],
            [-2, 0, 0.2],
            [-2, 1, 0.25],
            [-10, -1, 1],
            [-11, 1, 2],
            [-11, 1, 1],
            [-11, 1, 2],
            [-7, 2, 1],
  ]
  for l in lists:
    el = list_expand(l)
    if len(el) > 3:
      print("list {} expands to {}".format(l, el))
    else:
      print("list {} does not expand".format(l))
```